examples_tests/28.BSDFValidator/ShaderManager.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+
+MIT License
+
+Copyright (c) 2019 Achal Pandey
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+*/
+
+#include "ShaderManager.h"
+
+#include "SMaterial.h"
+#include "IGPUProgrammingServices.h"
+
+#include <iostream>
+#include <string>
+
+namespace irr
```

Comment:
do not put random example class in my engine's namespaces!

Reply:
Yeah, this was something I was kind of confused about.
Why do we put `BSDFValidatorApp` in `irr` namespace, then? I observed similar behavior in the BRDF Explorer.

Reply:
because I wansn't babysitting the person who wrote it.